docs/topics/media-pipeline.rst

```diff
@@ -197,6 +197,19 @@ For self-hosting you also might feel the need not to use SSL and not to verify S
     AWS_USE_SSL = False # or True (None by default)
     AWS_VERIFY = False # or True (None by default)
 
+
+Custom Amazon S3 headers
+''''''''''''''''''''''''
+
+Steps to pass Headers to scrapy FilesPipeline using AWS S3
+
+Configuration:
```

Comment:
These are still to paragraphs, and I see some minor issues of consistency with the rest of the documentation. What about:

```suggestion
To send custom headers to AWS S3 when using :class:`~scrapy.pipelines.files.FilesPipeline`
or :class:`~scrapy.pipelines.images.ImagesPipeline`, follow these steps:
```